### PR TITLE
Load a variable file for devices partition

### PR DESCRIPTION
**Pull request by WingkaiHo**
load device partition file in directory host_vars

1) if the user define host_vars/hostname.yml load the devices  partition on this file.
2) otherwise load host_vars/default.yml for default

### CODE REVIEW

File: infrastructure-playbooks/make-osd-partitions.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-# This playbook will make custom partition layout for your osd hosts.
+# This playbook will make custom  layout for your osd hosts.
```
Comment (leseb) [inline infrastructure-playbooks/make-osd-partitions.yml]: there is an extra space between 'custom' and 'layout'

File: infrastructure-playbooks/make-osd-partitions.yml [hunk 2]
```diff
@@ -30,6 +30,13 @@
   - "{{ osd_group_name }}"
 
   tasks:
+  
+  - name: Load a variable file for devices partition
```
Comment (leseb) [inline infrastructure-playbooks/make-osd-partitions.yml]: please remove the capital 'L'.

Comment (WingkaiHo) [inline infrastructure-playbooks/make-osd-partitions.yml]: OK

**Comment by WingkaiHo** [comment]
ok

**Comment by leseb** [comment]
jenkins test playbook syntax

**Comment by leseb** [comment]
still the same problem

File: infrastructure-playbooks/make-osd-partitions.yml [hunk 3]
```diff
@@ -26,6 +26,7 @@
     osd_group_name: osds
     journal_typecode: 45b0969e-9b03-4f30-b4c6-b4b80ceff106
     data_typecode: 4fbd7e29-9d25-41b8-afd0-062c0ceff05d
+    deviecs: []
```
Comment (leseb) [inline infrastructure-playbooks/make-osd-partitions.yml]: s/deviecs/devices/

**Comment by zhsj** [comment]
Maybe some late, why we make a task to combine host var and default var. IMHO, ansible itself takes care of it. You can define default var in group_vars/all and host var in host_vars/hostname

**Pull request merged**